docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -483,9 +483,9 @@ Current set of claims definitions:
 | sgx_config_svn             | uint16_t           | SGX           |Enclave configuration security version.                              |
 | sgx_isv_family_id          | uint8_t[16]        | SGX           |Enclave family ID.                                                   |
 | sgx_cpu_svn                | uint8_t[16]        | SGX           |Enclave CPU security version from sgx_report_t.                      |
-| sgx_pce_svn                | uint16_t           | SGX Remote    |PCE security version from quote.                                     |
-| sgx_qe_id                  | uint8_t[16]        | SGX Remote    |SGX Quoting Enclave ID, referred as QE_ID in SGX_ECDSA_QuoteGenReference_DCAP_API 3.1.4|
-| sgx_fmspc                  | uint8_t[6]         | SGX Remote    |SGX Family-Model-Stepping-Platform-Custom SKU from PCE certificate.  |
+| software_svn               | uint16_t           | SGX Remote    |PCE security version from quote.                                     |
+| platform_id                | uint8_t[16]        | SGX Remote    |SGX Quoting Enclave ID, referred as QE_ID in SGX_ECDSA_QuoteGenReference_DCAP_API 3.1.4|
+| hardware_family_id         | uint8_t[6]         | SGX Remote    |SGX Family-Model-Stepping-Platform-Custom SKU from PCE certificate.  |
```

Comment:
PR description says the intent is to be non-sgx specific but the category and descriptions say these are SGX specific.  This seems to be a contradiction.  Recommend updating the category and description to be non-SGX specific.

Reply:
updated category and comments.